### ISSUE TITLE
Clarify relationship between fbm and stochastic packages?

### ISSUE THREAD

**Issue by anntzer** [crf204/fbm#8]
Thanks for providing both `fbm` and `stochastic` for everyone to use :-)
Can you please clarify the relationship between the two packages?  I have the impression that fbm is basically a subset of stochastic, except that stochastic does not provide the Cholesky method (which could be ported there).  Am I missing anything?

**Comment by crflynn** [comment]
That's pretty much it. I wrote fbm first and later decided to expand to more processes. I created a second package, stochastic, with a more appropriate name. The stochastic package has better object naming and a better API, although there are a few things I would consider changing about it if/when I put time into it.

The Davies-Harte method is O(n log n), and falls back to Hosking, which is O(n^2), under conditions where Davies-Harte is not possible (when eigenvalues become negative). Cholesky's time-complexity is worse that Davies-Harte, so I didn't believe it was necessary, although it might be interesting for those looking to compare methods.
